en/docs/setup/deployment-guidelines-in-production.md

```diff
@@ -415,7 +373,7 @@ running the product.
 ### Hazelcast properties
 
 WSO2 products use
-[Hazelcast](https://docs.wso2.com/display/ADMIN44x/Clustering+Overview)
+[Hazelcast](../../admin-guide/clustering-Overview)
```

Comment:
'O' in overview should be changed to simple case.

Reply:
Fixed

Reply:
O in overview should be simple